packages/mobile/src/components/scrubber/Scrubber.tsx

```diff
@@ -150,7 +151,11 @@ export const Scrubber = ({
 
   return (
     <View style={styles.root}>
-      <Text style={styles.timestamp} weight='regular' numberOfLines={1}>
+      <Text
+        style={[styles.timestamp, { textAlign: 'right' }]}
```

Comment:
can we add textAlign to timestamp style?

Reply:
Yes but the right hand time stamp also uses the same style. So there would need to be a `timestampLeft`. I figured since it was just the alignment it would be ok to leave like this. What do you think?